tests/queries/0_stateless/03130_generate_snowflake_id.sql

```diff
@@ -0,0 +1,11 @@
+SELECT bitShiftLeft(toUInt64(generateSnowflakeID()), 52) = 0;
```

Comment:
That's the machine id, right (please add a code comment)? How is it guaranteed to be 0 after hashing the server uuid? I guess you need to incorporate the [server uuid itself](https://clickhouse.com/docs/en/sql-reference/functions/uuid-functions#serveruuid) into the comparison here.